src/index.ts

```diff
@@ -223,6 +226,22 @@ class LiskMigrator extends Command {
 			await migrateUserConfig();
 			cli.action.stop();
 		}
+
+		if (autoDownloadLiskCoreV4) {
+			cli.action.start('Installing lisk-core v4');
+			await installLiskCore();
+			cli.action.stop();
+		}
+
+		if (autoStartLiskCoreV4) {
+			cli.action.start('Starting lisk-core v4');
+			try {
+				await startLiskCore('PASS THE CONFIGURATION PATH', appVersion, { network: 'devnet' });
```

Comment:
Network can be resolved from `NETWORK_CONSTANT` instead of hard-coding.
For example: 
```
  const network = NETWORK_CONSTANT[info.networkIdentifier].name;
```

Reply:
Applied.